lib/matplotlib/backends/backend_tkagg.py

```diff
@@ -609,6 +609,9 @@ def destroy(*args):
             self.window.update()
         else:
             self.canvas.draw_idle()
+        # Raise the new window.
+        self.canvas.manager.window.attributes('-topmost', 1)
```

Comment:
http://stackoverflow.com/questions/1892339/how-to-make-a-window-jump-to-the-front